com.unity.UnityHub.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.unity.UnityHub.desktop</id>
+  <name>Unity Hub</name>
+  <developer_name>Unity Technologies</developer_name>
+  <summary>Real-time game engine</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://unity3d.com/</url>
+  <icon type="remote" height="256" width="256">https://raw.githubusercontent.com/kirbyfan64/flathub/unity/Unity.png</icon>
+  <description>
+    <p>Unity is used to create half of the world’s games. Our real time platform, powered by tools and services, offer incredible possibilities for game developers, and creators across industries and applications.</p>
+
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Unity Technologies.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://blogs.unity3d.com/wp-content/uploads/2018/09/image1-3.png</image>
+    </screenshot>
+  </screenshots>
+  <kudos>
+    <kudo>HiDpiIcon</kudo>
+  </kudos>
+  <categories>
+    <category>Development</category>
+  </categories>
+  <releases>
+    <release version="1.6.0" date="2019-03-07"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```

Comment:
The `none` values here aren’t wrong, but you can leave them out to increase readability if you want.